client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/style.scss

```diff
@@ -115,6 +117,11 @@ body.is-section-stepper {
 		.woo-confirm__info-section {
 			display: flex;
 			justify-content: center;
+			margin-top: 25vh;
+
+			@include break-small {
+				margin-top: 45vh;
```

Comment:
atleast in chrome, this didn't seem to be neccessary, but I did notice an awkward width as I dragged the window between mobile and desktop size, where the top margin was breaking the page layout. I think maybe we should set `margin-top: 0` on break-small?
<img width="437" alt="Screen Shot 2022-07-27 at 3 10 09 pm" src="https://user-images.githubusercontent.com/22446385/181166210-70a79021-7f87-4e5b-afdc-c7fdc3fc75b6.png">

Reply:
This was meant to fix spacing issue of the loading [indicator](https://github.com/Automattic/wp-calypso/blob/c04d343e7c51ec3bc0f180892ab388c554601996/client/landing/stepper/declarative-flow/internals/steps-repository/woo-confirm/index.tsx#L272)
I just found out the issue is caused by an [empty div](woo-confirm__info-section) with the same name of the loading wrapper, should we get rid of it instead?

Reply:
@bangank36 the empty div link seems to not work for me, could you check?

Reply:
Sorry I fixed the broken link. You can check again
Also screenshot is attached
<img width="1517" alt="Markup 2022-07-28 at 08 04 27" src="https://user-images.githubusercontent.com/10071857/181398365-14ad8079-fe89-4ecb-b24a-7c7fe524a241.png">

Reply:
I would recommend renaming the `woo-confirm__info-section` used to wrap `<LoadingEllipsis />` to something like `woo-confirm__loading-container`, then separate the CSS rules. For instance:

```
.site-setup.woo-confirm {
        .woo-confirm__loading-container,
        .woo-confirm__info-section {
            display: flex;
            justify-content: center;
        }

        .woo-confirm__loading-container {
            margin-top: 25vh;

            @include break-small {
                margin-top: 45vh;
            }

            .wpcom__loading-ellipsis {
                display: block;
                margin: 0 auto;
            }
        }
}
```

Reply:
Sorted!